src/NetAppFiles/NetAppFiles/Account/SetNetAppFilesAccount.cs

```diff
@@ -0,0 +1,90 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.NetAppFiles.Common;
+using Microsoft.Azure.Commands.NetAppFiles.Models;
+using Microsoft.Azure.Management.NetApp;
+using Microsoft.Azure.Management.NetApp.Models;
+using Microsoft.Azure.Commands.NetAppFiles.Helpers;
+using System.Collections.Generic;
+
+// Note:
+// Both set and Update need to exist
+// Patch of active directories can only alter the content
+// to remove the active directory a put is required
+
+namespace Microsoft.Azure.Commands.NetAppFiles.Account
+{
+    [Cmdlet(
+        "Set",
+        ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "NetAppFilesAccount",
+        SupportsShouldProcess = true,
+        DefaultParameterSetName = FieldsParameterSet), OutputType(typeof(PSNetAppFilesAccount))]
+    [Alias("Set-AnfAccount")]
+    public class SetAzureRmNetAppFilesAccount : AzureNetAppFilesCmdletBase
+    {
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The resource group of the ANF account")]
+        [ValidateNotNullOrEmpty]
+        [ResourceGroupCompleter()]
+        public string ResourceGroupName { get; set; }
```

Comment:
See: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/examples/set-cmdlet-example.md

You must implement the InoutObject and ResourceId parameter sets as wekl.